src/main/scala/GuessWhoGame/Board.scala

```diff
@@ -1,7 +1,116 @@
 package GuessWhoGame
 
-case class Board (Characters: List[Character]) {
+import scala.util.Random
+
+//values could be more domain specific, name/selecting could be within character case class, then able to test separately
+case class Board(characters: List[Character]) {
   def printCharacterNames(): Unit = {
-    Characters.foreach(Character => println(Character.name))
+    characters.foreach(Character => println(Character.name))
+  }
+
+  val selectedCharacter: Character = selectRandomCharacter(characters)
+  var remainingCharacters: List[Character] = characters
+
+  def printRemainingCharacters(): Unit = {
+    remainingCharacters.foreach(Character => println(Character.name))
+  }
```

Comment:
Not ideal to have a method returning a unit. Is there a way to get around this?

Reply:
We need to make sure a value is returned at the end of the method. This has been implemented across methods.